source/Octopus.Tentacle.Client/Scripts/Models/ExecuteScriptCommand.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Octopus.Tentacle.Contracts;
+
+namespace Octopus.Tentacle.Client.Scripts.Models
+{
+    public class ExecuteScriptCommand
```

Comment:
I wonder if the parent of this and ExecuteKubernetesScriptCommand, ought to be an interface of the minimum things we think any of these commands must have.

This concrete class might become `ExecuteShellScriptCommand`

Reply:
yeah, that's a good idea 👍
I might make it an abstract just due to how much shared properties there are, but 👍

Reply:
done in [8886e99](https://github.com/OctopusDeploy/OctopusTentacle/pull/845/commits/8886e99b2755537ab48270f1a28141562f3ef682)